libs/shared/data-access/work-api/src/lib/models/schema-tree/schema-tree.model.ts

```diff
@@ -106,7 +106,8 @@ export class SchemaTreeDefinitionModel implements SchemaModel<ISchemaTreeDefinit
         root.label = current.name;
 
         // Assign type for filtering schema keys by type
-        root['type'] = this.getTypeString(current.type, current.contentType);
+        root['type'] = current.type;
+        if (current.contentType) root['contentType'] = current.contentType;
```

Comment:
You should put this in brackets.
Single line is good for simple returns, for anything more complicated it becomes harder to read

```suggestion
        if (current.contentType) {
          root['contentType'] = current.contentType;
        }
```